src/applications/vaos/tests/appointment-list/components/RequestedAppointmentsList.unit.spec.jsx

```diff
@@ -19,6 +27,14 @@ const initialState = {
 };
 
 describe('VAOS <RequestedAppointmentsList>', () => {
+  beforeEach(() => {
+    mockFetch();
+    MockDate.set(getTimezoneTestDate());
+  });
+  afterEach(() => {
+    resetFetch();
+    MockDate.reset();
+  });
   it('should show va request', async () => {
     const startDate = moment.utc();
```

Comment:
Is `.utc()` okay here? I noticed you removed it in the file below.

Reply:
Yeah, it shouldn't matter. I removed it from the other file when debugging a failure and forgot to add it back.